src/hotspot/share/classfile/stackMapTable.hpp

```diff
@@ -116,8 +114,22 @@ class StackMapReader : StackObj {
   char* _code_data;
   int32_t _code_length;
 
-  // information get from the attribute
-  int32_t  _frame_count;       // frame count
+  // information from the attribute
+  int32_t  _frame_count;
+
+  // Number of frames parsed
+  int32_t  _parsed_frame_count;
+
+  // Previous and current frame buffers
+  StackMapFrame* _pre_frame;
```

Comment:
If you've touched all of the instances of pre_frame, can you rename it prev_frame ?  frame_buf could also be current_frame if you've changed lines that it appears on.

Reply:
I noticed that frame_buf is unused and as far as I can tell it isn't needed. "prev_frame" is definitely clearer so I'll switch to that.